frigate/__main__.py

```diff
@@ -1,17 +1,28 @@
 import faulthandler
+import logging
 import threading
 
 from flask import cli
 
 from frigate.app import FrigateApp
 
-faulthandler.enable()
 
-threading.current_thread().name = "frigate"
+def main() -> None:
+    faulthandler.enable()
+    logging.basicConfig(
+        level=logging.INFO,
+        handlers=[
+            # Prevent other code from registering root handlers accidentally
+            logging.NullHandler()
```

Comment:
this doesn't fix the issue, logs are still duplicated

```
INFO:frigate.app:Starting Frigate (0.15.0-68e49af3)
[2024-09-17 06:40:50] frigate.app                    INFO    : Starting Frigate (0.15.0-68e49af3)
INFO:frigate.util.config:Checking if frigate config needs migration...
[2024-09-17 06:40:50] frigate.util.config            INFO    : Checking if frigate config needs migration...
INFO:frigate.util.config:frigate config does not need migration...
[2024-09-17 06:40:50] frigate.util.config            INFO    : frigate config does not need migration...
INFO:peewee_migrate.logs:Starting migrations
[2024-09-17 06:40:50] peewee_migrate.logs            INFO    : Starting migrations
INFO:peewee_migrate.logs:There is nothing to migrate
[2024-09-17 06:40:50] peewee_migrate.logs            INFO    : There is nothing to migrate
INFO:frigate.app:Recording process started: 31352
INFO:frigate.app:Review process started: 31353
```

Reply:
I'm seeing the same thing.

Reply:
Blindness intensifies... I was looking at the snippet for a good minute before I spotted the duplicates.

`import onvif` runs before `main()`, so I didn't fix anything.

Reply:
Now I did.